### DISCUSSION TITLE
Pos vs Neg

### DISCUSSION THREAD

**Discussion by evokercodex** [tylerjamesf/MissingNo#5] [General]
https://github.com/phantomfirefox/MissingNo/blob/de5ba12f17a9f32e195789a9467bd1c4e44c2212/Pokemon%20Natures.csv#L3

I think you mentioned values could positive / negatively affect other values? Should their be an crosswalk to determine how it impacts other attributes?

**Comment by tylerjamesf** [comment]
I think we discussed this on the other section but probably have to stay away from negatives unless its add/subtract since it will cause negative final values and break

**Reply by evokercodex** [reply]
Yup - we did! We can "close" this discussion as they are represented as a percent multiplier and will adjust appropriately. i.e. add 10% (1.1) or reduce by 10% (.9).
